src/components/WidgetVideo/WidgetVideo.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import PropTypes from 'prop-types'
```

Comment:
Is this part of this PR?

Reply:
no, I'm not sure how that was duplicated in there. I've removed the file now after some investigation and testing that it all still works without this file